src/guards/isUploadedImage.ts

```diff
@@ -1,14 +1,18 @@
+import { isPlainObject } from 'is-plain-object';
 import { isUploadedFile } from './isUploadedFile';
 import type { UploadedImage } from '../types';
 
 function isNonEmptyString(value: unknown): boolean {
     return typeof value === 'string' && value.length > 0;
 }
 
-export function isUploadedImage(file: any): file is UploadedImage {
+export function isUploadedImage(
+    file: Record<string, any> | undefined | null,
+): file is UploadedImage {
     return (
         typeof file === 'object' &&
-        typeof file.original_height === 'number' &&
+        isPlainObject(file) &&
+        typeof file?.original_height === 'number' &&
```

Comment:
why is this necessary? Isn't `isPlainObject` already checking for `file` object to be not null?